sdk/lib/core/iterable.dart

```diff
@@ -915,6 +933,17 @@ class _GeneratorIterable<E> extends ListIterable<E> {
   static int _id(int n) => n;
 }
 
+class _FromIteratorIterable<E> extends Iterable<E> {
```

Comment:
Could rename it `withIteratorIterable` to match the constructor name. 
(It's private, so nobody will know.)